templates/patterns/list_item/pattern-list-item--variant-highlight.html.twig

```diff
@@ -11,6 +11,7 @@
       path: url,
       type: "standalone"
     },
-    image: image
+    image: image,
+    description: detail,
```

Comment:
```suggestion
    description: detail|smart_trim(length),
```
There is `length` parameter in the interface of List item pattern - "Used to trim the length of the 'Detail' field (optional).". I think we should support it here also. If length is not provided full description is shown. 
See another variants of List item patterns.